src/server/api/routes/vessel-reports.router.js

```diff
@@ -31,4 +31,16 @@ router.get("/:id", authorizeUser(ROLES.SUPER_USER), function(req, res, next) {
     .catch(next);
 });
 
+// ENDPOINT: /api/vessel-reports/:vessel-report-id/suggested-route
+router.get(
+  "/:vessel_report_id/suggested-route",
```

Comment:
Suggested routes should be plural `suggested_routes` to indicate that we are getting _all_ suggested_routes matching a given `vessel_id`.

Reply:
made it plural